applications/FluidDynamicsApplication/custom_processes/embedded_skin_visualization_process.cpp

```diff
@@ -923,6 +950,15 @@ const Vector EmbeddedSkinVisualizationProcess::SetDistancesVector(ModelPart::Ele
     return nodal_distances;
 }
 
+const Vector EmbeddedSkinVisualizationProcess::SetEdgeDistancesExtrapolatedVector(ModelPart::ElementIterator ItElem)
```

Comment:
```suggestion
const inline Vector EmbeddedSkinVisualizationProcess::SetEdgeDistancesExtrapolatedVector(const Element& rElem)
```
(about passing a reference instead of an iterator, I'd prefer it even though it is not consistent with the other methods. Again a minor detail).

Reply:
Stupid question, but how do I get the element from its index?

Reply:
From the iterator you are passing you can just do `*it_elem`.

Reply:
Thanks!